_posts/2022-06-13-connecting-the-pods.md

```diff
@@ -0,0 +1,291 @@
+It is designed to take decisions and implement changes as quickly and as close to the customer as possible. 
+Whereas the traditional company employs processes that can be seen as recipes or a series of steps that need to be executed, the podular organization is structured as a net, distributing the workload based on goals instead of allocating individual steps or stages. 
+And the pods are not necessarily part of the same organization or company but could extend across several to couple its inherent benefits with more economy-of-scale oriented benefits.
+
+This gives a number of advantages based on its lattice-like structure, exchanging consistency and efficiency for flexibility and creativity:
+* Pods are flexible: changing the structure and purpose is easy and fast.
+* Pods are allowed to fail as the network will provide redundancy by adjusting the purpose of similar pods.
+* Pods can scale up fast by adding additional pods and letting them immediately contribute.
+
+This type of flexibility cannot exist in a vacuum. 
+It needs a proper support system (or platform) to make it happen, necessitating a shared infrastructure for all available and future pods, such as for example a proper cloud adoption. 
+Although it might seem quite a daunting task to put in place, this shared infrastructure offers numerous benefits:
+* More traffic brings more customers and in turn more business.
+* It allows for a focus on differentiators, making it possible to prioritize certain services and products that are “hot” at the moment, only to switch to others when the need arises.
+* The commonality between these pods (common technological standards and shared company culture) generates trust with customers who can see a rise in the overall quality of all services and products offered.
+
+As with all organizations, the connected company needs to continue to evolve. 
+This is done by learning, and the connected company sees learning as a growth spiral called successive approximation. 
+The connected company takes actions, which leads to feedback and discovery based on the outcome of such actions. 
+Once these are collected, you reflect upon how to change the way actions are taken to improve their result. 
+When compared to more traditional ways of learning such as the Double-Loop Learning, to me, this seems like the reflection part is portrayed as a bit too straightforward. 
+If the results of your actions are the only thing to reflect upon, the reframing of the mindset is missed.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-06-13-connecting-the-pods/learning.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+To go a bit more into detail on this type of learning: Professor Chris Argyris of Harvard Business School identifies three possible levels of learning in an organization, which he coined single-loop, double-loop, and deutero learning. 
```

Comment:
link or reference to this?

Reply:
same. no particular source. He mentions it in a lot of his work